src/order/closure.lean

```diff
@@ -73,10 +73,25 @@ def mk' (f : α → α) (hf₁ : monotone f) (hf₂ : ∀ x, x ≤ f x) (hf₃ :
   le_closure' := hf₂,
   idempotent' := λ x, le_antisymm (hf₃ x) (hf₁ (hf₂ x)) }
 
+/-- Constructor for a closure operator using the weaker minimality axiom: `x ≤ f y → f x ≤ f y`. -/
```

Comment:
```suggestion
/-- Convenience constructor for a closure operator using the weaker minimality 
axiom: `x ≤ f y → f x ≤ f y`, which can sometimes be easier in practice. -/
```

Reply:
And something similar for `mk₃` too, to explain a little bit why these are nice to have